app/models/helper.py

```diff
@@ -0,0 +1,16 @@
+from flask import jsonify, abort, make_response
+
+def get_one_obj_or_abort(cls, obj_id):
```

Comment:
Great helper function, but I don't think it should be in the models folder since it's route specific.